master/buildbot/clients/tryclient.py

```diff
@@ -349,23 +350,23 @@ def readConfig(self):
 
     def parseConfig(self, res):
         self.config = {}
-        for l in res.split("\n"):
+        for l in res.split(b"\n"):
             if l.strip():
-                parts = l.strip().split("=", 2)
+                parts = l.strip().split(b"=", 2)
                 if len(parts) < 2:
                     parts.append('true')
                 self.config[parts[0]] = parts[1]
         return self.config
 
     def parseTrackingBranch(self, res):
         # If we're tracking a remote, consider that the base.
-        remote = self.config.get("branch." + self.branch + ".remote")
-        ref = self.config.get("branch." + self.branch + ".merge")
+        remote = self.config.get(b"branch." + self.branch + b".remote")
```

Comment:
the expression is repeated here, a single variable would be better to avoid c&p errors

Reply:
oops, i misread the code :(